crmsh/bootstrap.py

```diff
@@ -186,6 +195,61 @@ def init_sbd_manager(self):
         from .sbd import SBDManager
         self.sbd_manager = SBDManager(self)
 
+    def detect_platform(self):
+        """
+        Detect platform
+        """
+        self.is_s390 = "390" in os.uname().machine
+        if not self.is_s390:
+            self.cloud_type = utils.detect_cloud()
+
+    def determine_profile_type(self):
+        """
+        Determine profile type
+        """
+        if self.is_s390:
+            profile_type = self.S390_PROFILE_NAME
+        elif self.cloud_type:
+            profile_type = self.cloud_type
+        else:
+            return None
```

Comment:
This looks a typical place where an Exception is raised, but this `None` option can be good in any case